.changeset/cute-taxis-double.md

```diff
@@ -0,0 +1,24 @@
+---
+'@backstage/backend-defaults': minor
+'@backstage/integration': minor
+---
+
+**Attention:** Support for Bitbucket Cloud's API token was added. App password is deprecated (no new creation from September 9, 2025) and will be removed on June 9, 2026.
+
+API token (recommended):
+
+```yaml
+integrations:
+  bitbucketCloud:
+    - username: user@domain.com
+      token: my-token
+```
+
+App password (deprecated):
+
+```yaml
+integrations:
+  bitbucketCloud:
+    - username: user
+      appPassword: my-secret
+```
```

Comment:
Shoot, forgot to mention we should also update the docs here as well: https://github.com/backstage/backstage/blob/master/docs/integrations/bitbucketCloud/locations.md